templates/home/class_schedule.html

```diff
@@ -0,0 +1,143 @@
+{% extends 'sidebar.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Class Schedule {% endblock %}
+
+{% block content-sidebar %}
+
+<!-- CLASS SCHEDULE -->
+
+<div class="dash-schedule col-md-8 col-lg-9 mt-3">
+    <a href="" class="d-inline-block mb-2 mb-md-0">&#x27f3; Refresh Page</a>
+    <h3 class="text-center">Class Schedule</h3>
+        <div class="row mt-5 justify-content-center align-items-center">
+            <div class="col-md-3 mb-3">
+                <label for="sch-day">Day</label>
+                <select name="day" class="custom-select d-block w-100" id="sch-day">
+                    <option value="">Select Day</option>
+                    {% for day, day_name in days %}
+                    <option value="{{ day }}">{{ day_name }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+            <div class="col-md-3 mb-3">
+                <label for="sch-section">Section</label>
+                <select name="section" class="custom-select d-block " id="sch-section">
+                    <option value="">Select Section</option>
+                    {% for section in sections %}
+                    <option value="{{ section }}">{{ section }}</option>
```

Comment:
You should use `section.section_id` in value attribute and `section.name` in the HTML part.